### ISSUE TITLE
Exclude retweets from timeline

### ISSUE THREAD

**Issue by maxpel** [DocNow/twarc#485]
Hi!

In the documentation I did not find a flag to enable exclusion of retweets (or replies) from twarc2 timeline.
This could be useful for people that want only original tweets by the user to be able to get more data with the monthly tweet cap of the Academic API.

According to https://developer.twitter.com/en/docs/twitter-api/tweets/timelines/migrate/standard-to-twitter-api-v2, this is pretty straightforward by just including `exclude=retweets` (or `exclude=retweets,replies`) in the API call. See the example:

`curl --request GET --url 'https://api.twitter.com/2/users/XXX/tweets?tweet.fields=attachments,author_id,created_at,entities,geo,id,in_reply_to_user_id,lang,possibly_sensitive,public_metrics,referenced_tweets,source,withheld&user.fields=created_at,description,entities,id,location,name,profile_image_url,protected,public_metrics,url,username,verified,withheld&expansions=author_id,referenced_tweets.id,referenced_tweets.id.author_id,entities.mentions.username,attachments.media_keys&media.fields=duration_ms,height,preview_image_url,public_metrics,url,width&exclude=retweets' --header 'Authorization: Bearer XXX' -o file.json`

It could be an option to timeline:

```
twarc2 timeline --exclude-retweets
twarc2 timeline --exclude-replies
```

Thanks for your great work with twarc2!

Best wishes,
maxpe

**Comment by edsu** [comment]
The --exclude-retweets and --exclude-replies options have been added to the timeline and timelines subcommands. It hsan't been released yet to PyPI though. I'll add a note here when it has (should be a day or so).

**Comment by edsu** [comment]
Ok, this functionality has been released as part of v2.2.0 on PyPI so you should be able to upgrade with `pip install --upgrade twarc`.
